play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukCookieBanner.scala.html

```diff
@@ -15,12 +15,14 @@
  *@
 
 @this(govukButton: GovukButton)
-
 @(params: CookieBanner)
 @import params._
 
-<div class="@toClasses("govuk-cookie-banner ", classes)" data-nosnippet role="region" aria-label="@ariaLabel.getOrElse("Cookie banner")"@if(hidden){ hidden}@toAttributes(attributes)>@for(message <- messages) {
-    <div class="@toClasses("govuk-cookie-banner__message govuk-width-container", message.classes)"@message.role.mapNonEmpty {role => role="@role"} @toAttributes(message.attributes) @if(message.hidden){ hidden}>
+<div class="@toClasses("govuk-cookie-banner", classes)" data-nosnippet role="region" aria-label="@{ariaLabel match {
+    case Some(NonEmptyString(str)) => str
+    case _ => "Cookie banner"
+   }}"@if(hidden){ hidden}@toAttributes(attributes)>@for(message <- messages) {
```

Comment:
damn mind bending twirl `@if(hidden){ hidden}` being "output 'hidden' literally" rather than the value of the hidden variable :oof: